src/collection-loader/index.js

```diff
@@ -35,22 +34,25 @@ import enhanceCollection from "../enhance-collection"
 import feed from "./feed"
 import cache from "./cache"
 import description from "./description"
+import validator from "./validator"
 
 let timeout
 
 module.exports = function(input) {
 
-  const query = loaderUtils.parseQuery(this.query)
+  const query = {
+    ...this.options.statinamic && this.options.statinamic.collection,
+    ...loaderUtils.parseQuery(this.query),
```

Comment:
I this we should validate both line independently, this will allow to provide more accurates errors.